flink-python/pyflink/fn_execution/coder_impl_fast.pyx

```diff
@@ -262,9 +263,10 @@ cdef class IterableCoderImpl(LengthPrefixBaseCoderImpl):
         self._separated_with_end_message = separated_with_end_message
 
     cpdef encode_to_stream(self, value, LengthPrefixOutputStream output_stream):
-        for item in value:
-            self._field_coder.encode_to_stream(item, self._data_out_stream)
-            self._write_data_to_output_stream(output_stream)
+        if value:
```

Comment:
What's the purpose of this change?

Reply:
It is a hotfix for IterableCoderImpl when input value is a None after we reconstructiing coders